metadata.yaml

```diff
@@ -6,6 +6,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
```

Comment:
This is correct but I think we didn't document it in release tasks. Can you please add it there?

Reply:
Document it, thanks